### ISSUE TITLE
Add new lesson to current queue while reviewing

### ISSUE THREAD

**Issue by bmironer** [davidsansome/tsurukame#126]
While reviewing the current queue and done/remaining counters are not updated when a new session is due. So you might think you’re almost done and find out you really have 50 more...

**Comment by hesch** [comment]
This is consistent with the WaniKani website. It also doesn't update the count while reviewing and only shows a little banner in the review overview screen to tell you that there are more reviews.

**Comment by turtlemaster19** [comment]
I agree that this works as intended. The app's notification banner will show the total amount of Reviews actually available, if you just wanted to access that number.
